client/me/profile-gravatar/index.jsx

```diff
@@ -12,80 +11,36 @@ import { connect } from 'react-redux';
 import Animate from 'components/animate';
 import Gravatar from 'components/gravatar';
 import { recordGoogleEvent } from 'state/analytics/actions';
-import { isEnabled } from 'config';
```

Comment:
Cool, opened a new PR with enabling it in production using the feature flag: https://github.com/Automattic/wp-calypso/pull/16604